src/dashboard/widget_modal.py

```diff
@@ -263,6 +276,24 @@ def toggle_modal(
                         dash.no_update,
                         False,
                         True,
+                        False,
+                        dash.no_update,
+                    )
+                if (
+                    date.today() < pd.to_datetime(start_date).date()
```

Comment:
Rather than pd.to_datetime, can we use the datetime library since this is already included and used?

You can convert the strings to a datetime by datetime.datetime.strptime(date, "%Y-%m-%d")
See here for reference: https://stackoverflow.com/questions/466345/converting-string-into-datetime

Reply:
Changed to use datetime library